assets/src/components/map/controls/layersControl.tsx

```diff
@@ -0,0 +1,103 @@
+import Leaflet from "leaflet"
+import React, { useEffect, useState } from "react"
+import ReactDOM from "react-dom"
+import { TileLayer, useMap, useMapEvents } from "react-leaflet"
+import { joinClasses } from "../../../helpers/dom"
+import { TileType, tilesetUrlForType } from "../../../tilesetUrls"
+import { MapLayersIcon } from "../../../helpers/icon"
+
+export const LayersControl = ({
+  tileType,
+  setTileType,
+}: {
+  tileType: TileType
+  setTileType: (tileType: TileType) => void
+}): JSX.Element | null => {
+  const map = useMap()
+  const portalParent = map
+    .getContainer()
+    .querySelector(".leaflet-control-container .leaflet-top.leaflet-right")
+  const [portalElement, setPortalElement] = useState<HTMLElement | null>(null)
+
+  const [showLayersList, setShowLayersList] = useState(false)
+
+  useEffect(() => {
+    if (!portalParent || !portalElement) {
+      setPortalElement(document.createElement("div"))
+    }
+
+    if (portalParent && portalElement) {
+      portalElement.className = "c-map__layer-control"
+      portalParent.append(portalElement)
+      Leaflet.DomEvent.disableClickPropagation(portalElement)
```

Comment:
issue: based on the [built-in leaflet layers control code](https://github.com/Leaflet/Leaflet/blob/b6b56f6e707142c177fad2f67827a5007e56736a/src/control/Control.Layers.js#L186-L187), I think `disableScrollPropagation` should be added.

```suggestion
      Leaflet.DomEvent.disableClickPropagation(portalElement)
      Leaflet.DomEvent.disableScrollPropagation(portalElement)
```

Reply:
Upon checking this out in other controls, it seems like none of the other ones do this, so I'd be fine with leaving it if Design or other Devs are fine with this.